test/controllers/project_category_controller_test.exs

```diff
@@ -1,47 +1,25 @@
 defmodule CodeCorps.ProjectCategoryControllerTest do
-  use CodeCorps.ApiCase
-
-  alias CodeCorps.Category
-  alias CodeCorps.Project
-  alias CodeCorps.ProjectCategory
-  alias CodeCorps.Repo
-
-  @attrs %{}
-
-  defp build_payload, do: %{ "data" => %{"type" => "project-category", "attributes" => %{}}}
+  use CodeCorps.ApiCase, resource_name: :project_category
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
-      conn = get conn, project_category_path(conn, :index)
+      path = conn |> project_category_path(:index)
+      conn = conn |> get(path)
       assert json_response(conn, 200)["data"] == []
     end
 
     test "filters resources on index", %{conn: conn} do
-      arts = insert(:category, name: "Arts")
-      society = insert(:category, name: "Society")
       technology = insert(:category, name: "Technology")
-
       project = insert(:project)
-      project_category_1 = insert(:project_category, project: project, category: arts)
-      project_category_2 = insert(:project_category, project: project, category: society)
+      [project_category_1, project_category_2] = insert_pair(:project_category)
```

Comment:
Same as [comment](https://github.com/code-corps/code-corps-api/pull/425/files#r86388037), while this is good, it could be rewritten in one line as 

```Elixir
[project_category_1, project_category_2 | _] = insert_list(3, : project_category_1)
```

We don't really need to pre-insert the category, or the project. The factory should do that part for us.